test/triplex/triplex_test.exs

```diff
@@ -35,15 +37,17 @@ defmodule TriplexTest do
   end
 
   test "create/2 must return a error if the tenant already exists" do
+    prefix = Triplex.config().tenant_prefix
+    pg_msg = "ERROR 42P06 (duplicate_schema) schema #{prefix}lala already exists"
+    ms_msg = "(1007): Can't create database '#{prefix}lala'; database exists"
+
     assert {:ok, _} = Triplex.create("lala", PGTestRepo)
 
-    assert {:error, "ERROR 42P06 (duplicate_schema) schema \"lala\" already exists"} =
-             Triplex.create("lala", PGTestRepo)
+    assert {:error, pg_msg} = Triplex.create("lala", PGTestRepo)
```

Comment:
The warning you are getting here is because this line is not actually checking if `Triplex.create("lala", PGTestRepo)` returns `{:error, pg_msg}`, but actually assigning `pg_msg` again, and you never used it... I would change it to:

```suggestion
    assert Triplex.create("lala", PGTestRepo) == {:error, pg_msg}
```

Reply:
As above